kafka/src/main/java/io/specmesh/kafka/Clients.java

```diff
@@ -158,25 +160,34 @@ private static String buildJaasConfig(final String userName, final String passwo
                 + "\";";
     }
 
+    /**
+     * @deprecated use {@link #schemaRegistryClient(String, String, String)}
+     */
+    @Deprecated(forRemoval = true)
```

Comment:
deprecated, as calling a method to create a sr client when `srEnabled = false` just doesn't make sense.